<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background: #ffffff;
    color: #222;
    font-family: Segoe UI, Arial, sans-serif;
    font-size: 11pt;
    background-image: 
    background-repeat: no-repeat;
    background-position: center;
    background-attachment: fixed;
    background-size: cover;
}
QGroupBox {
    border: 1px solid #cccccc;
    border-radius: 8px;
    margin-top: 10px;
    background: #fafafa;
}
QPushButton {
    padding: 6px 16px;
    border-radius: 6px;
    background: #f5f5f5;
    border: 1px solid #cccccc;
}
QPushButton:checked, QPushButton:pressed {
    background: #e0e0e0;
}
QTableWidget {
    background: #fff;
    alternate-background-color: #f7f7f7;
    gridline-color: #e0e0e0;
}
QHeaderView::section {
    background: #e0e0e0;
    font-weight: bold;
    font-size: 12pt;
    border: 1px solid #cccccc;
}
QLineEdit, QTextEdit, QSpinBox {
    background: #fff;
    border: 1px solid #cccccc;
    border-radius: 4px;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>461</width>
     <height>101</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>6</width>
     <height>6</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Titillium Web</family>
     <underline>true</underline>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Yapılacaklar Listesi&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>171</y>
     <width>89</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>ekle</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Downloads/images.png</normaloff>../../Downloads/images.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>417</x>
     <y>171</y>
     <width>50</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>sil</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>307</x>
     <y>171</y>
     <width>103</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>tamamla</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>621</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>40</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>111</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>111</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Öncelik&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Başlık</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Açıklama</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>111</width>
     <height>85</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>474</x>
     <y>174</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_dusuk">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Düşük</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_orta">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>80</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Orta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_yuksek">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>80</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Yüksek</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
